pkg/discovery/processor/turbo_policy_processor.go

```diff
@@ -26,13 +26,20 @@ func (p *TurboPolicyProcessor) ProcessTurboPolicies() {
 	turboSloScalings, err := p.ClusterScraper.GetAllTurboSLOScalings()
 	if err != nil {
 		glog.Warningf("Failed to list SLOHorizontalScales: %v.", err)
-		return
 	}
-	if len(turboSloScalings) == 0 {
-		glog.V(2).Info("There is no SLOHorizontalScale resource found in the cluster.")
+
```

Comment:
Please generalize and put logging in a separate function.  This should not need to change if new policies are supported in future.

Reply:
will address this in https://vmturbo.atlassian.net/browse/OM-98560.